src/overlays/actors/ovl_Demo_Im/z_demo_im.c

```diff
@@ -206,33 +209,34 @@ void func_80984F94(DemoIm* this, GlobalContext* globalCtx) {
     func_80034A14(&this->actor, &this->unk_2D4, kREG(17) + 0xC, 4);
 }
 
-void func_80985018(DemoIm* this, GlobalContext* globalCtx) {
+void DemoIm_UpdateBgCheckInfo(DemoIm* this, GlobalContext* globalCtx) {
     Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 75.0f, 30.0f, 30.0f, 5);
 }
 
-s32 func_80985060(DemoIm* this) {
+s32 DemoIm_UpdateSkelAnime(DemoIm* this) {
     return SkelAnime_Update(&this->skelAnime);
 }
 
-s32 func_80985080(GlobalContext* globalCtx) {
+s32 DemoIm_IsCsStateIdle(GlobalContext* globalCtx) {
     if (globalCtx->csCtx.state == CS_STATE_IDLE) {
         return 1;
+    } else {
+        return 0;
```

Comment:
`DemoIm_IsCsStateIdle`
```suggestion
        return true;
    } else {
        return false;
```